docs/examples/affinity/cookie/README.md

```diff
@@ -1,27 +1,29 @@
-# Sticky Session
+# Sticky sessions
 
-This example demonstrates how to achieve session affinity using cookies
+This example demonstrates how to achieve session affinity using cookies.
 
 ## Deployment
 
-Session stickiness is achieved through 3 annotations on the Ingress, as shown in the [example](ingress.yaml).
+Session stickiness can be configured using the following annotations. Make sure to prefix their names with `nginx.ingress.kubernetes.io/` when setting them.
 
-|Name|Description|Values|
+|Name <small>without prefix</small>|Description|Value|
 | --- | --- | --- |
-|nginx.ingress.kubernetes.io/affinity|Sets the affinity type|string (in NGINX only ``cookie`` is possible|
-|nginx.ingress.kubernetes.io/session-cookie-name|Name of the cookie that will be used|string (default to INGRESSCOOKIE)|
-|nginx.ingress.kubernetes.io/session-cookie-expires|The value is a date as UNIX timestamp that the cookie will expire on, it corresponds to cookie Expires directive|number of seconds|
-|nginx.ingress.kubernetes.io/session-cookie-max-age|Number of seconds until the cookie expires that will correspond to cookie `Max-Age` directive|number of seconds|
+|affinity|Type of the affinity, used to enable it|string (NGINX only supports `cookie`)|
+|session-cookie-name|Name of the cookie that will be created|string (defaults to `INGRESSCOOKIE`)|
+|session-cookie-hash|Type of hash that will be used to generate the cookie value|`sha1`/`md5`/`index` (defaults to `sha1`)|
```

Comment:
this does not exist anymore https://github.com/kubernetes/ingress-nginx/pull/3743